stetho/src/main/java/com/facebook/stetho/inspector/screencast/ScreencastDispatcher.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.screencast;
+
+import android.app.Activity;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Matrix;
+import android.graphics.RectF;
+import android.os.Handler;
+import android.os.HandlerThread;
+import android.os.Looper;
+import android.util.Base64;
+import android.util.Base64OutputStream;
+import android.view.View;
+import com.facebook.stetho.common.LogUtil;
+import com.facebook.stetho.inspector.elements.android.ActivityTracker;
+import com.facebook.stetho.inspector.jsonrpc.JsonRpcPeer;
+import com.facebook.stetho.inspector.protocol.module.Page;
+
+import java.io.ByteArrayOutputStream;
+
+public final class ScreencastDispatcher {
+  private static final long FRAME_DELAY = 200l;
+
+  private final Handler mMainHandler = new Handler(Looper.getMainLooper());
+  private final BitmapFetchRunnable mBitmapFetchRunnable = new BitmapFetchRunnable();
+  private final ActivityTracker mActivityTracker = ActivityTracker.get();
+  private final EventDispatchRunnable mEventDispatchRunnable = new EventDispatchRunnable();
+  private final RectF mTempSrc = new RectF();
+  private final RectF mTempDst = new RectF();
+
+  private boolean mIsRunning;
+  private Handler mBackgroundHandler;
+  private JsonRpcPeer mPeer;
+  private HandlerThread mHandlerThread;
+  private Bitmap mBitmap;
+  private Canvas mCanvas;
+  private Page.StartScreencastRequest mRequest;
+  private ByteArrayOutputStream mStream;
+  private Page.ScreencastFrameEvent mEvent = new Page.ScreencastFrameEvent();
+  private Page.ScreencastFrameEventMetadata mMetadata = new Page.ScreencastFrameEventMetadata();
+
+  public ScreencastDispatcher() {
+  }
+
+  public void startScreencast(JsonRpcPeer peer, Page.StartScreencastRequest request) {
+    LogUtil.d("Starting screencast");
+    mRequest = request;
+    mHandlerThread = new HandlerThread("Screencast Thread");
+    mHandlerThread.start();
+    mPeer = peer;
+    mIsRunning = true;
+    mStream = new ByteArrayOutputStream();
+    mBackgroundHandler = new Handler(mHandlerThread.getLooper());
+    mMainHandler.postDelayed(mBitmapFetchRunnable, FRAME_DELAY);
+  }
+
+  public void stopScreencast() {
+    LogUtil.d("Stopping screencast");
+    mBackgroundHandler.post(new CancellationRunnable());
+  }
+
+  private class BitmapFetchRunnable implements Runnable {
+    @Override
+    public void run() {
+      getScreenBitmap();
```

Comment:
We're calling a 'get' method but then we're not using the return value ... and there are no other calls to this method, so there isn't really a good reason for this choice of verb. This method should probably be called `updateScreenBitmap`, and have its return type changed to void.